app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from __future__ import annotations
```

Comment:
I thin this import redundant here

Reply:
I have the following error without this import

app/main.py:7: in
def calculate_team_total_rating(team: list[Player]) -> int:
E TypeError: 'type' object is not subscriptable

I can avoid using this import if I use the type annotation that looks like this
(team: list)